src/components/ConstraintSelect.tsx

```diff
@@ -7,15 +7,25 @@ interface IConstraintSelectProps {
    * @param constraint package version constraint
    */
   constraint: string | null;
+  onUpdate?: (value: string) => void;
 }
 
-export const ConstraintSelect = ({ constraint }: IConstraintSelectProps) => {
+export const ConstraintSelect = ({
+  constraint,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
either `return void` here or remove the `no-empty-function` rule from our eslint config